packages/components/src/tools-panel/tools-panel-header/component.js

```diff
@@ -27,35 +41,87 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 		return null;
 	}
 
+	const defaultItems = Object.entries( menuItems?.default || {} );
+	const optionalItems = Object.entries( menuItems?.optional || {} );
+
 	return (
 		<h2 { ...headerProps } ref={ forwardedRef }>
 			{ labelText }
 			{ hasMenuItems && (
-				<DropdownMenu icon={ moreVertical } label={ labelText }>
+				<DropdownMenu
+					icon={ moreVertical }
+					label={ labelText }
+					menuProps={ { style: { minWidth: '200px' } } }
+				>
 					{ ( { onClose } ) => (
 						<>
-							<MenuGroup label={ __( 'Display options' ) }>
-								{ Object.entries( menuItems ).map(
-									( [ label, isSelected ] ) => {
-										return (
-											<MenuItem
-												key={ label }
-												icon={ isSelected && check }
-												isSelected={ isSelected }
-												onClick={ () => {
-													toggleItem( label );
-													onClose();
-												} }
-												role="menuitemcheckbox"
-											>
-												{ label }
-											</MenuItem>
-										);
-									}
-								) }
-							</MenuGroup>
+							{ !! defaultItems?.length && (
+								<MenuGroup>
+									{ defaultItems.map(
+										( [ label, hasValue ] ) => {
+											const icon = hasValue
+												? minus
+												: check;
+
+											const itemLabel = hasValue
+												? sprintf(
+														// translators: %s: The name of the control being reset e.g. "Padding".
+														__( 'Reset %s' ),
+														label
+												  )
+												: undefined;
```

Comment:
Nit: should this piece of logic also be moved to the `getAriaLabel` function? It seems weird to use that function only for optional items

Reply:
It does but so did adding extra conditionals within `getAriaLabel` along with passing extra args. In the end, I went with the simplest version of both the function and its use. I don't feel strongly about it, so happy to change as requested.